src/lifecycles/load.js

```diff
@@ -47,6 +47,10 @@ export function toLoadPromise(app) {
           return app;
         }
 
+        if (appOpts.devtools && appOpts.devtools.overlays) {
+          app.devtools.overlays = {...app.devtools.overlays, ...appOpts.devtools.overlays}
+        }
```

Comment:
shallow merge the default overlays and the specified overlays coming from the application.

Applications can now optionally export a devtools object. So for contacts-ui it would look something like this:
```js
export const devtools = {
  overlays: {
    selectors: [
      '#contacts-ui .cps-nav-content',
      '#contacts-ui .cps-topnav-secondary',
      '.addContactParcelClass',
    ]
  }
}
```